src/ButtonGroup/index.js

```diff
@@ -0,0 +1,149 @@
+import React, { PureComponent, Children, cloneElement } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { transparentize } from 'polished';
+
+import Button from '../Button';
+
+const { func, node, string } = PropTypes;
+
+/**
+ * Button Group
+ *
+ * This component is in charge of displaying
+ * a button group
+ *
+ * @param {node} children // Anything that can be rendered: numbers, strings, elements or an array (or fragment).
+ * @param {string} className // Add a text aside in the select next the selected value.
+ * @param {function} onChange // Callback function called when a button is clicked.
+ * @param {string} defaultActiveButton // Specifies the initial state: witch button is selected.
+ *
+ * @return {jsx}
+ */
+
+class ButtonGroup extends PureComponent {
+  /**
+   * PropTypes Validation.
+   */
+  static propTypes = {
+    children: node,
+    className: string,
+    onChange: func,
+    defaultActiveButton: string,
+  };
+
+  /**
+   * Default props.
+   */
+  static defaultProps = {
+    children: null,
+    className: '',
+    onChange: () => {},
+    defaultActiveButton: '',
+  };
+
+  /** Internal state. */
+  state = {
+    activeButton: this.props.defaultActiveButton, // eslint-disable-line react/destructuring-assignment
+  };
+
+  /**
+   * updateActiveButton.
+   * Tell to the store witch button is activated
+   * @param {string} name
+   */
+  updateActiveButton = (name, hasClick) => {
```

Comment:
1. **Documentation summary**

> `Tell to the store witch button is activated`

Why are you talking about? Probably more something like:

> `Handles click on a child button. In addition, notifies the parent about a click on a button child.`

What do you think?

2. **Documentation parameters**

Missing `hasClick` param in documentation.

3. **Parameter naming**

I'd rename `hasClick` to `onClick` since I'd expect `hasClick` to be only a boolean, not a callback function.